src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -6,36 +6,33 @@
 
 import seedu.address.model.AddressBook;
 import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.person.*;
```

Comment:
I think it would be better to keep the single import statements rather than use a wildcard import.